### ISSUE TITLE
inject mac80211_hwsim frames to real WiFi network

### ISSUE THREAD

**Issue by bam80** [vanhoefm/libwifi#1]
Hello,
is it possible to somehow connect mac80211_hwsim simulated interfaces to a real ones?

**Comment by raghava-2002** [comment]
> Hello, is it possible to somehow connect mac80211_hwsim simulated interfaces to a real ones?

Yes, it is possible to connect the mac80211_hwsim interfaces to the real interfaces (USB WiFi interface cards ex: ar9271) make the real interface card as the AP then build a bridge to connect the mac80211_hwsim interfaces as a client, and vice-versa is also possible.
